CONTRIBUTING.md

```diff
@@ -265,12 +244,12 @@ The test suite can be run from a command line or via your IDE. You can run `make
 run all unit tests. Another way to run the test suite is to use
 [**tox**](https://tox.readthedocs.io/en/latest/#). For more information about using tox please
 refer to
-[Terra CONTRIBUTING](https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#test)
+[Qiskit CONTRIBUTING](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md#test)
 Test section. However please note Nature does not have any
-[online tests](https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#online-tests)
+[online tests](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md#online-tests)
```

Comment:
This link does not exist any more. I am not even sure what this was originally referring to.

Reply:
I admit I did not check over these links - it is the same across the app repos as the contributing came from a common source. Its likely that since this was originally done, based on what Terra had, that these not longer exist. I believe this was around if the ibm provider was used - some tutorials had an optional section that you could enable. I do not recall what might have been in terra long ago. We could leave this for now - rather than make it different from the other app repos. I really do not know if/how this file might change - I just wanted to change any refs from Terra and removed some text that was not how things are done in order to remove any confusion that may have caused going forwards,